mocca-client/src/main/java/com/paypal/mocca/client/MoccaClient.java

```diff
@@ -3,21 +3,13 @@
 import feign.AsyncClient;
 import feign.AsyncFeign;
 import feign.Feign;
-import feign.Request;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-import java.time.Duration;
 import java.util.HashSet;
 import java.util.Set;
-import java.util.concurrent.TimeUnit;
-import java.util.function.Supplier;
 
 /**
- * Applications are supposed to create an interface,
- * extending this one, to define their GraphQL client API.
- * Each GraphQL operation can be defined using Mocca
- * annotations under {@link com.paypal.mocca.client.annotation}.
+ * Applications are supposed to create an interface, extending this one, to define their GraphQL client API. Each
+ * GraphQL operation can be defined using Mocca annotations under {@link com.paypal.mocca.client.annotation}.
  * <br>
  * <br>
```

Comment:
Even though you remove most of it, please, search the word timeout. You will find a few mentions of it in javadoc.

Reply:
I believe I got it from all javadoc/test/markdown.  Please double check.  I'm not the best at such things:)

Reply:
There is one more, line 40 ;-)

Reply:
This comment is still pending

Reply:
AFAIK, line 40 is as it should be.  If you would like something different, please use the 'suggest' feature.